src/plugins/analysis/users_and_passwords/install.py

```diff
@@ -26,11 +26,11 @@ class UsersAndPasswordsInstaller(AbstractPluginInstaller):
     def install_system_packages(self):
         super().install_system_packages()
 
-        lshw_p = subprocess.run('lshw -c display'.split(), stdout=PIPE, stderr=PIPE, check=True)
+        lshw_process = subprocess.run('lshw -c display'.split(), stdout=PIPE, stderr=PIPE, check=True)
         opencl_pkgs = []
-        if lshw_p.stdout == 'NVIDIA':
+        if lshw_process.stdout == 'NVIDIA':
             opencl_pkgs = ['nvidia-opencl-dev']
-        elif lshw_p.stdout == 'AMD':
+        elif lshw_process.stdout == 'AMD':
```

Comment:
It seems this did not work in the past because it should either also use a `universal_newlines=True` or compare the `stdout` with byte strings